Applications/Spire/Resources/Resources.qrc

```diff
@@ -109,5 +109,6 @@
         <file>Icons/text_effects/underline-active.svg</file>
         <file>Icons/text_effects/underline-hover.svg</file>
         <file>Icons/text_effects/underline-inactive.svg</file>
+        <file>Icons/arrow-down-grey.svg</file>
```

Comment:
As a suggestion, would it be helpful to order these alphabetically? Is there a logical order to these resources?

Reply:
I typically edit this file through Qt Creator, so it appears the resources just get added in chronological order.

I don't find modifying the resources file cumbersome, but it might be a good idea to add more prefixes if you're expecting to add a lot of resources in the future, so the resources look like:

Icons/BookView/file1.png
Icons/SecurityInput/file2.png
Icons/UI/file3.png
etc...

Reply:
Okay, something to think about for another task then.